### PR TITLE
net-p2p/retroshare:a add 0.6.7 with new USE flag

### PR DESCRIPTION
**Pull request by gentoo-setan**
This is a continuation of #36386 and my third try to upgrading retroshare.

A new USE flag has been added and new version is officialy here
---

Please check all the boxes that apply:

- [X] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [X] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [X] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [X] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Comment by parona-source** [comment]
Probably better to bump to 0.6.7.2 as  0.6.7 and 0.6.7.1 appears to have been skipped (despite been tagged).

"Official 0.6.7 release"
https://github.com/RetroShare/RetroShare/releases/tag/v0.6.7.2

https://github.com/RetroShare/RetroShare/compare/v0.6.7...v0.6.7.2

File: net-p2p/retroshare/retroshare-0.6.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+		x11-libs/libX11
+		x11-libs/libXScrnSaver
+	)
+	libupnp? ( net-libs/libupnp:= )
+	miniupnp? ( net-libs/miniupnpc:= )
+	sqlcipher? ( dev-db/sqlcipher )
+	!sqlcipher? ( dev-db/sqlite:3 )
+	plugins? (
+		media-libs/speex
+		<media-video/ffmpeg-5
```
Comment (juippis) [inline net-p2p/retroshare/retroshare-0.6.7.ebuild:49]: I do think this won't live for long but it's easy to mask when the time to remove ffmpeg-4 comes. Just know that I'm a bit reluctant in adding new versions that still depend on ffmpeg-4.

Comment (juippis) [inline net-p2p/retroshare/retroshare-0.6.7.ebuild:49]:
```suggestion
	plugins? (
		media-libs/speex
		media-libs/speexdsp
		<media-video/ffmpeg-5
```

**Review by juippis — APPROVED** [review]
Thanks, I'll merge and squash the commits!

File: net-p2p/retroshare/retroshare-0.6.7.ebuild (hunk 1 above)
Comment (gentoo-setan) [reply net-p2p/retroshare/retroshare-0.6.7.ebuild:49]: I agree, but:

```
compiling with the 'plugins' use flag enabled may trigger multiple errors about not declared functions/macros in plugins/VOIP/gui/VideoProcessor.cpp when >=media-video/ffmpeg-5 is installed in the system:

avcodec_register_all
AV_CODEC_CAP_TRUNCATED
AV_CODEC_FLAG_TRUNCATED
avcodec_encode_video2
avcodec_decode_video2
av_free_packet

The aforementioned functions/macros have all been removed from FFmpeg by upstream at some point after v4.5:

https://github.com/FFmpeg/FFmpeg/tree/c361fa9e215cf21fb0e5b3395d675aec3d216719
```
-- mehw ([link](https://bugs.gentoo.org/927207#c5))

Comment (gentoo-setan) [reply net-p2p/retroshare/retroshare-0.6.7.ebuild:49]: Makes sense.
Resolving this since PR's closed?

Comment (juippis) [reply net-p2p/retroshare/retroshare-0.6.7.ebuild:49]: Probably forgot when I started looking into qtscript dep. I'll add the dep.

Comment (o0nd7ots) [reply net-p2p/retroshare/retroshare-0.6.7.ebuild:49]: A mistake then.

Glad that it;s fixed!

**Pull request closed without merging**